nzedb/Releases.php

```diff
@@ -199,7 +200,50 @@ public function getBrowseRange($cat, $start, $num, $orderBy, $maxAge = -1, $excl
 	{
 		$orderBy = $this->getBrowseOrder($orderBy);
 
-		$qry = sprintf(
+        if ($this->hideuploadednzb) {
+            $qry = sprintf(
+                "SELECT r.*,
+                    CONCAT(cp.title, ' > ', c.title) AS category_name,
+                    CONCAT(cp.id, ',', c.id) AS category_ids,
+                    df.failed AS failed,
+                    rn.releaseid AS nfoid,
+                    re.releaseid AS reid,
+                    v.tvdb, v.trakt, v.tvrage, v.tvmaze, v.imdb, v.tmdb,
+                    tve.title, tve.firstaired
+                FROM
+                (
+                    SELECT r.*, g.name AS group_name
+                    FROM releases r
+                    LEFT JOIN groups g ON g.id = r.group_id
+                    WHERE r.nzbstatus = %d
+                    AND r.name NOT LIKE '%%.nzb\" yEnc'
```

Comment:
This may not be the best choice of filtering.  Doing any kind of LIKE may be fine with 100k releases.  But some of the bigger guys have millions of releases.